fatiando/tests/gravmag/test_transform.py

```diff
@@ -198,3 +198,29 @@ def test_laplace_from_potential():
     assert np.all(np.abs(laplace) <= 1e-10), \
         "Max: {} Mean: {} STD: {}".format(
             laplace.max(), laplace.mean(), laplace.std())
+
+
+def test_potential_field_tilt_values():
+    "gravmag.transform tilt returns sane values, between -90 and 90 degrees"
+    model = [Prism(-500, 500, -500, 500, 0, 2000, {'magnetization': 200})]
+    shape = (300, 300)
+    x, y, z = gridder.regular([-10000, 10000, -10000, 10000], shape, z=-100)
+    inc, dec = -30, 15
+    potential = prism.tf(x, y, z, model, inc, dec)
+    pole = transform.reduce_to_pole(x, y, potential, shape, inc, dec,
+        sinc=inc, sdec=dec)
+    tilt_angle = transform.tilt(x, y, pole, shape)
+    assert max(np.degrees(tilt_angle)) < 90, \
+        "Maximum tilt greater than 90: {}".format(max(np.degrees(tilt_angle)))
+    assert min(np.degrees(tilt_angle)) > -90, \
+        "Minimum tilt less than -90: {}".format(min(np.degrees(tilt_angle)))
+
+
+def test_potential_field_tilt_works():
+    "gravmag.transform tilt returns sane values"
+    model = [Prism(-1000, 1000, -500, 500, 0, 2000, {'magnetization': 3200})]
```

Comment:
Another thing that will be deprecated is passing in a scalar magnetization. This is implicitly saying that there is only induced magnetization. We'll be changing the `gravmag` functions to accept only vector magnetization so that this is explicit.